wandb/board/ui/src/containers/Runs.js

```diff
@@ -262,6 +278,41 @@ class Runs extends React.Component {
               on="click"
               position="bottom left"
             />
+            <Button
+              floated="right"
+              disabled={this.props.data.selectedRuns.length === 0}
+              onClick={e => {
+                e.preventDefault();
+
+                this.setState({
+                  showConfirm: true,
+                  confirmText: 'Are you sure you would like to hide these runs? You can reverse this later by removing the "hidden" label.',
+                  confirmButton: `Hide ${this.props.data.selectedRuns.length} run(s)`,
+                  handleConfirm: e => {
```

Comment:
Why do these need to be in state?

Reply:
Added a todo so it's at least clear that code is my fault.